loader_hub/github_repo/base.py

```diff
@@ -5,25 +5,50 @@
 The documents are either the contents of the files in the repository or
 the text extracted from the files using the parser.
 """
+import os
 import asyncio
 import base64
 import binascii
-import enum
 import logging
-import os
 import pathlib
 import tempfile
+import enum
+import sys
 from typing import Any, Callable, List, Optional, Tuple
 
 from llama_index.readers.base import BaseReader
 from llama_index.readers.file.base import DEFAULT_FILE_EXTRACTOR
-from llama_index.readers.llamahub_modules.github_repo.github_client import (
-    BaseGithubClient, GitBranchResponseModel, GitCommitResponseModel,
-    GithubClient, GitTreeResponseModel)
-from llama_index.readers.llamahub_modules.github_repo.utils import (
-    BufferedGitBlobDataIterator, get_file_extension, print_if_verbose)
 from llama_index.readers.schema.base import Document
 
+
+if "pytest" in sys.modules:
```

Comment:
ah i see. this is to get unit tests to pass?

Reply:
yeah, Unfortunately I couldn't find a nicer way to do this